themes/classic/_dev/js/product.js

```diff
@@ -137,8 +137,11 @@ $(document).ready(() => {
       buttonup_class: 'btn btn-touchspin js-touchspin',
       min: parseInt($quantityInput.attr('min'), 10),
       max: 1000000,
+      mousewheel: false,
```

Comment:
Is it disabled only here on purpose?

Reply:
You were right, I added it while trying some things, but it's not needed